packages/vite-plugin/src/utils.ts

```diff
@@ -20,7 +20,13 @@ export function getProjects(project: string | string[]): ProjectData[] {
     const {name, dir} = path.posix.parse(url);
     const metaFile = `${name}.meta`;
     const metaData = getMeta(path.join(dir, metaFile));
-    const data = {name: metaData?.name ?? name, fileName: name, url};
+    const filePath = [...dir.split('/').slice(2), name].join('/');
```

Comment:
We don't know if people use `src` to store their projects, someone could use a `videoA/project.ts`, `videoB/project.ts` structure with no `src`.
I think we should use the entire path here and not strip the `src`:
```suggestion
    const filePath = path.posix.join(dir, name);
```